### PR TITLE
add styles and filter button

### PR DESCRIPTION
**Pull request by YinkTech**
Complete React - Redux Capstone Project 

- Live link - https://yinktech.github.io/Stock-Market
- Loom Video - https://www.loom.com/share/b62c2d2bfbb64f54b42faeab763cbe03

### CODE REVIEW

File: src/components/Filter.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <Dropdown.Item eventKey="2" className='border fw-bold fs-3'>2</Dropdown.Item>
+    <Dropdown.Item eventKey="3" className='border fw-bold fs-3'>3</Dropdown.Item>
+    <Dropdown.Item eventKey="4" className='border fw-bold fs-3'>4</Dropdown.Item>
+    <Dropdown.Item eventKey="5" className='border fw-bold fs-3'>5</Dropdown.Item>
+    <Dropdown.Item eventKey="6" className='border fw-bold fs-3'>6</Dropdown.Item>
+    <Dropdown.Item eventKey="7" className='border fw-bold fs-3'>7</Dropdown.Item>
+    <Dropdown.Item eventKey="8" className='border fw-bold fs-3'>8</Dropdown.Item>
+  </Dropdown.Menu>
+</Dropdown>
+);
```
Comment (soieducation) [inline src/components/Filter.js]: - [x] Using the filter breaks the app and returns the error shown in the video [Error video](https://www.loom.com/share/03f80029f77649e992339b4b995e42c3)
 
- [x] Not quite sure what the functionality of the filter is, kindly explain in your video description

**Review by soieducation — CHANGES_REQUESTED** [review]
Hi @YinkTech, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Details.js [hunk 2]
```diff
@@ -55,10 +55,12 @@ const Details = () => {
                       Key <span className='def-color'>Statistics</span> 
                     </h3>
                     <div className='text-center p-2'>
-                      <div className='mb-4 mx-3 market-row p-3 shadow text-success'>
+
+                    <div className='mb-4 mx-3 market-row p-3 shadow text-success'>
```
Comment (soieducation) [inline src/components/Details.js:59]: - The design of the page is awesome👍

**Review by soieducation — APPROVED** [review]
Hi @YinkTech, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/components/Details.js (hunk 2 above)
Comment (YinkTech) [reply src/components/Details.js:59]: Thanks so much, ma'am🕺🏽🥳💃, it makes me happy.

**Pull request merged**